docs/pages/platform/webhooks.mdx

```diff
@@ -972,7 +972,10 @@ type ThreadNotificationEvent = {
     userId: string;
     threadId: string;
     inboxNotificationId: string;
+    // Date representing the time when the webhook event was created.
     createdAt: string;
+    // Date representing the  time when the notification was created.
```

Comment:
Also applies to the same comments below and in `/liveblocks-node/src/webhooks.ts`.

```suggestion
    // Date representing the time when the notification itself was created.
```